processing/src/main/java/org/apache/druid/query/aggregation/AggregatorFactory.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.druid.segment.ColumnInspector;
 import org.apache.druid.segment.ColumnSelectorFactory;
 import org.apache.druid.segment.column.ColumnType;
-import org.apache.druid.segment.column.ColumnTypeFactory;
+//import org.apache.druid.segment.column.ColumnTypeFactory;
```

Comment:
Is that getting accepted in checkstyle?

Reply:
i think there is no code style for processing tht why not failed